src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
+
+public abstract class Bot {
+
+    abstract String reactTo(String command, String[] args);
+    abstract String onAdd();  // what bot does when it is first added
+    public String getName()
```

Comment:
Does any logic rely on the fact that `getName()` returns the class name instead of some other name? If so, what happens if a subclass overrides this function? If that would break something, maybe mark this function `final` so subclasses can't override it?